tools/build/README.md

```diff
@@ -30,6 +30,8 @@ The default behaviors of the scripts (without parameters) assume that the direct
   - package.json
   - tsconfig.json
 
+If the directory structure fo your package varies from this, define either RUSHSTACK_FILE_ERROR_BASE_FOLDER or NODE_PROJECT_ROOT_DIRECTORY environment variables as the path to the directory containing your package.json file.
```

Comment:
1. "of" not "fo"
2. Suggest users use the NODE_PROJECT* env variable, and maybe note parenthetically that the RUSHSTACK_FILE_* env var is deprecated.

Reply:
Oh was looking at outdated... for not of, got it.